src/Sylius/Bundle/ShopBundle/templates/product_review/index/main/list/review/content.html.twig

```diff
@@ -0,0 +1,5 @@
+{% set review = hookable_metadata.context.review %}
+
+<div class="mb-2">
+    {{ review.comment }}
```

Comment:
Would be nice to have a `content` hook extracted like the `footer` and `header` ones.